### PR TITLE
Packaging updates

### PR DESCRIPTION
**Pull request by ehildenb**
-   Get packaging message from `INSTALL.md` instead of hard-coded string in `Jenkinsfile`.
-   Flesh out build and runtime dependencies for Arch, Ubuntu, and Debian Buster.
-   Switch `PKGBUILD` to a `git` VCS based build, and follow guidelines here about how to do that: https://wiki.archlinux.org/index.php/VCS_package_guidelines
-   Explicitly run `install-rust` in setup phase for packaging, as a fresh-vm has not done this.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+### Ubuntu/Debian
+
+Install the package with (`X.Y.Z` is version number, `ID` is platform identifier):
+
+```sh
+sudo apt install ./kframework_X.Y.Z_amd64_ID.deb
+```
+
+On Debian Stretch, you also must first enable stretch-backports by running:
```
Comment (dwightguth) [inline INSTALL.md]: we support buster now, not stretch, and on buster this is not true.

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+Installing Release Builds
+=========================
+
+These instructions explain how to download, install, and build the K packages.
+Current supported systems are:
+
+-   Arch Linux
+-   Ubuntu Bionic (18.04)
+-   Ubuntu Xenial (16.04)
```
Comment (dwightguth) [inline INSTALL.md]: we do not support xenial anymore.

**Review by dwightguth — CHANGES_REQUESTED** [review]
I'm not sure I like this git stuff either. When we do a stable release, it's going to be based on the source tarball that doesn't even have a .git directory, so git submodule update will fail. let's leave source control to source control and the packaging system for building and packaging.

I also have a couple other comments.

File: INSTALL.md (hunk 1 above)
Comment (ehildenb) [reply INSTALL.md]: Removed.

File: INSTALL.md (hunk 2 above)
Comment (ehildenb) [reply INSTALL.md]: Updated.

**Review by dwightguth — APPROVED** [review]

**Pull request merged**